include/TrustWalletCore/TWEthereumChainID.h

```diff
@@ -22,6 +22,7 @@ enum TWEthereumChainID {
     TWEthereumChainIDThunderToken = 18,
     TWEthereumChainIDTomoChain = 88,
     TWEthereumChainIDXDai = 100,
+    TWEthereumChainIDDEXON = 237,
```

Comment:
@alejandro-isaza @hewigovens do we need chain ID property? what I realized that we can get chain ID from the network and it's not required to get from the hardcoded values.